css-layout-classes/styles.css

```diff
@@ -0,0 +1,27 @@
+.container {
+  max-width: 800px;
+  margin: 0, auto;
```

Comment:
This syntax is incorrect. There should be no commas. It should just be `margin: 0 auto`